sql-plugin/src/main/301until330-all/scala/com/nvidia/spark/rapids/shims/GpuTypeShims.scala

```diff
@@ -46,4 +47,9 @@ object GpuTypeShims {
    * @return the cuDF type if the Shim supports
    */
   def toRapidsOrNull(t: DataType): DType = null
+
+  def supportCsvRead(dt: DataType) : Boolean = false
+
+  def csvRead(cv: ColumnVector, dt: DataType): ColumnVector =
```

Comment:
```suggestion
  def csvRead(cv: cudf.ColumnVector, dt: DataType): cudf.ColumnVector =
```
Otherwise, it will conflict with the PR https://github.com/NVIDIA/spark-rapids/pull/4926 who imports the `org.apache.spark.sql.vectorized.ColumnVector`